tweetflask.py

```diff
@@ -40,14 +50,16 @@ def get_tweet(jsondata, user, mood):
         return tweetdata
     for tweets in jsondata['TweetData']:
         if user['UserID'] == tweets['UserID']:
-            if mood is None or mood is not None and mood == tweets['Mood']:
+            if not mood or mood.lower() == tweets['Mood'].lower():
                 response_data = {}
-                response_data['id'] = user['UserID']
-                response_data['type'] = "string"
+                response_data['id'] = tweets['TweetID']
+                response_data['type'] = "tweet"
```

Comment:
Sometimes you use double quote (") but sometimes you use single quote ('). It's better to keep your code consistent by sticking on either one of them.